From eb84ad5656b17436200dc1ee6cb6d12605ac8e0c Mon Sep 17 00:00:00 2001
From: Auke Kok <auke-jan.h.kok@intel.com>
Date: Tue, 5 Nov 2019 12:55:38 -0800
Subject: [PATCH] build without libx264

v2: undo 2 changes in original patch that make changes to UI with no
effect on behavior

Signed-off-by: Chelsea Mafrica <chelsea.e.mafrica@intel.com>
---
 plugins/CMakeLists.txt          | 1 -
 plugins/obs-x264/CMakeLists.txt | 2 +-
 2 files changed, 1 insertion(+), 2 deletions(-)

diff --git a/plugins/CMakeLists.txt b/plugins/CMakeLists.txt
index 90bea35be273..d602468e06fd 100644
--- a/plugins/CMakeLists.txt
+++ b/plugins/CMakeLists.txt
@@ -175,7 +175,6 @@ else()
 endif()
 
 add_subdirectory(image-source)
-add_subdirectory(obs-x264)
 add_subdirectory(obs-libfdk)
 add_subdirectory(obs-ffmpeg)
 add_subdirectory(obs-outputs)
diff --git a/plugins/obs-x264/CMakeLists.txt b/plugins/obs-x264/CMakeLists.txt
index 196c5ab7ae53..1147cc6bb4a2 100644
--- a/plugins/obs-x264/CMakeLists.txt
+++ b/plugins/obs-x264/CMakeLists.txt
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 3.22...3.25)
 
 legacy_check()
 
-find_package(Libx264 REQUIRED)
+find_package(Libx264)
 
 if(NOT TARGET OBS::opts-parser)
   add_subdirectory("${CMAKE_SOURCE_DIR}/deps/opts-parser" "${CMAKE_BINARY_DIR}/deps/opts-parser")
-- 
2.45.1

